pkg/cloudprovider/vsphere/route/routeprovider.go

```diff
@@ -115,7 +116,7 @@ func (p *routeProvider) CreateRoute(ctx context.Context, clusterName string, nam
 	nodeName := string(route.TargetNode)
 	klog.V(6).Infof("Creating static route for node %s", nodeName)
 
-	nodeIP, err := p.getNodeIPAddress(nodeName, IsIPv4(route.DestinationCIDR))
+	nodeIP, err := p.getNodeIPAddress(nodeName, util.IsIPv4(route.DestinationCIDR))
```

Comment:
move `IsIPv4` into a common folder 'util' (unit tests added) so vsphere and vsphere-paravirtual can both use